ModelBundle/DataFixtures/MongoDB/LoadStatusDataFunctional.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\DataFixtures\MongoDB;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\ModelBundle\Document\Status;
+use OpenOrchestra\ModelBundle\Document\TranslatedValue;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraFunctionalFixturesInterface;
+
+/**
+ * Class LoadStatusDataFunctional
+ */
+class LoadStatusDataFunctional extends AbstractFixture implements OrderedFixtureInterface, OrchestraFunctionalFixturesInterface
```

Comment:
you can factorize LoadStatusDataFunctional and LoadStatusData, method  loadStatus and generateTranslatedValue is duplicated